Tasks/Barkun/Task_4/Task_4_word.py

```diff
@@ -0,0 +1,103 @@
+# 1. Прочитать содержимое файла text.txt
+# 2. Дать пользователю ввести с клавиатуры параметр "максимальное количество символов в строке", который должен быть больше 35
+# 3. Отформатировать текст с учётом максимального количества символов как ограничения на длину строки, при этом если слово целиком не умещается в строке, 
+# оно должно быть перенесено на следующую, а отступы между словами равномерно увеличены (по аналогии с функцией "Выровнять по ширине" текстовых редакторов)
+# 4. Записать получившийся текст в новый файл и оповестить об этом пользователя.
+# (модуль textwrap использовать нельзя)
+
+print ('\nGood day!\nI suggest you format the following text:\n')
+
+with open("text.txt", encoding='utf-8') as text:
+    text = text.read() 
```

Comment:
reading the whole file in memory is not the best approach since it can be very big